config/manager/manager.yaml

```diff
@@ -39,17 +39,5 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 9440
-          initialDelaySeconds: 15
-          periodSeconds: 20
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 9440
-          initialDelaySeconds: 5
-          periodSeconds: 10
```

Comment:
lets keep the manager  probes in place and we can test if we see the errors again.

Reply:
I think whatever was added by My PR should be removed. that way it will be easy to rule out any other reason.